File: 3.py
import sys

# Функция для переворота числа и удаления ведущих нулей
def revers_num(num):
    revers = 0
    while num != 0:  # Переворачиваем число
        revers = revers * 10 + num % 10
        num = num // 10  # Целочисленное деление
    return revers

def main():
    
    try:
        n = int(input())
    except ValueError:
        print("Количество элементов должно быть положительным и целым")
        sys.exit(1)  # Завершаем программу с ошибкой
        
    # Проверка на корректность ввода
    if n <= 0:
        print("Количество элементов должно быть положительным и целым")
        sys.exit(1)
    else:
        for _ in range(n):
            try:
                num = int(input())
            except ValueError:
                print("Числа должны быть положительным и целым")
                sys.exit(1)
                
            # Проверка на корректность ввода
            if num <= 0:
                print("Числа должны быть положительным и целым")
                sys.exit(1)
            else:
                # Переворачиваем число и выводим
                print(revers_num(num), end=" ") 

    sys.exit(0)

if __name__ == "__main__":
    main()

File: 3.cs
using System;

class Program
{
    // Функция для переворота числа и удаления ведущих нулей
    static int revers_num(int num)
    {
        int revers = 0;
        while (num != 0)  // Переворачиваем число
        {
            revers = revers * 10 + num % 10;
            num /= 10;
        }
        return revers;
    }

    static int Main()
    {
        Console.OutputEncoding = System.Text.Encoding.UTF8;
        int n;
        string input = Console.ReadLine();
        if (!int.TryParse(input, out n))
        {
            Console.WriteLine("Количество элементов должно быть положительным и целым");
            return 1;
        }

        // Проверка на корректность ввода
        if (n <= 0)
        {
            Console.WriteLine("Количество элементов должно быть положительным и целым");
            return 1;
        }
        else
        {
            for (int i = 0; i < n; ++i)
            {
                input = Console.ReadLine();
                int num;
                if (!int.TryParse(input, out num))
                {
                    Console.WriteLine("Количество элементов должно быть положительным и целым");
                    return 1;
                }

                // Проверка на корректность ввода
                if (num <= 0)
                {
                    Console.WriteLine("Числа должны быть положительным и целым");
                    return 1;
                }
                else
                {
                    // Переворачиваем число и выводим с пробелом
                    Console.Write(revers_num(num) + " ");
                }
            }
        }

        return 0;
    }
}
